.github/workflows/tests.yml

```diff
@@ -14,50 +15,53 @@ jobs:
       AWS_REGION: ${{ secrets.AWS_REGION }}
       EC2_SECRET: ${{ secrets.EC2_PWD }}
       S3_BACKEND_KEY_NAME: tf-state-${{ github.sha }}
+    outputs:
+      runner: ${{ steps.determine-runner.outputs.runner }}
     steps:
+      - id: determine-runner
+        name: Determine runner
+        run: if [ -z $AWS_ACCESS_KEY_ID ]; then echo "::set-output name=runner::ubuntu-latest"; else echo "::set-output name=runner::${{ github.sha }}"; fi;
       - name: checkout code
+        if: ${{ env.AWS_ACCESS_KEY_ID }}
         uses: actions/checkout@v2.1.0
       - name: setup node
+        if: ${{ env.AWS_ACCESS_KEY_ID }}
         uses: actions/setup-node@v2
         with:
           node-version: '14'
       - uses: hashicorp/setup-terraform@v1
+        if: ${{ env.AWS_ACCESS_KEY_ID }}
         with:
           terraform_version: 0.12.25
-          #- name: Declare some variables
-          # id: vars
-          #shell: bash
-          #run: |
-          # echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
-          #echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
       - name: rewrite tf state key
         id: tf-state-key
+        if: ${{ env.AWS_ACCESS_KEY_ID }}
         working-directory: ./infrastructure
         run: sed -i 's/BRANCH_ACTOR_KEY/${{ env.S3_BACKEND_KEY_NAME }}/g' infra.tf
       - name: Terraform Init
+        if: ${{ env.AWS_ACCESS_KEY_ID }}
         id: init
         working-directory: ./infrastructure
         run: terraform init
-        continue-on-error: false
```

Comment:
We should keep this unless it's false by default

Reply:
i removed it because github reported it as an unexpected input https://github.com/galacticcouncil/HydraDX-node/actions/runs/793403551. failing is default behavior anyway